README.md

```diff
@@ -99,7 +95,7 @@ If it's necessary (because the exercise is too long), no need to implement all t
 #### Requirements
 We initialize the folder [demonstrator](./demonstrator), but it's empty.
 
-There is no specific requirement for the demonstrator. \
+There is no specific requirement nor technical restriction for the demonstrator. \
```

Comment:
⚠️ this is what we agreed on, but let's be ready to get solutions that use framework (React for sure)